tests/ut_suite_manager/ut_suite_manager.emptySuitePath.sql

```diff
@@ -0,0 +1,33 @@
+set termout off
+create or replace package tst_empty_suite_path as
+  --%suite
+  --%suitepath
+
+  --%test
+  procedure test#1;
+end;
+/
+
+create or replace package body tst_empty_suite_path as
+  procedure test#1 is begin ut.expect(1).to_equal(1); end;
+end;
+/
+
+set termout on
+
+declare
+  l_objects_to_run ut_suite_items;
+begin
+
+  --act
+  l_objects_to_run := ut_suite_manager.configure_execution_by_path(ut_varchar2_list('tst_empty_suite_path'));
+
+  if ut_expectation_processor.get_status = ut_utils.tr_success then
```

Comment:
I don't understand how will this test ever fail, as there is no expectation executed before status is checked.